src/main/antlr4/Manifold.g4

```diff
@@ -89,14 +92,23 @@ lvalue:
   ;
 
 // TODO: declarations as expressions
-expression: /* declaration | */ rvalue;
+expression:
+  rvalue
+  // | declaration
+  ;
+
+statement:
+    expression #ExpressionStatment
+    | 'import' STRING_VALUE #ImportStatement
```

Comment:
As Lucas was mentioning, do we need to make imports separate statements? It would seem to me that we could use imports as rvalues.

Something like:
`HalfAdder = import HalfAdder from '../digital'`